frontend/packages/core/src/stories/buttongroup.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import type { Meta } from "@storybook/react";
+
+import type { ButtonGroupProps } from "../button";
+import { ButtonGroup } from "../button";
+
+export default {
+  title: "Core/Button Group",
```

Comment:
For the Note components I nested the panel under [Notes](https://deploy-preview-607--clutch-components.netlify.app/?path=/story/core-feedback-note-panel--primary) but I think I like this more. What do you think?

Reply:
Initially thought to organize it by:
```
Buttons
-- Button
-- Button Group
```
I looked at some Storybook examples, and I think it's a little easier to find/see the variety of components when the stories aren't too nested. For example, I like this [All Components](https://5d559397bae39100201eedc1-gyadsqdnlc.chromatic.com/?path=/story/playground-playground--playground) heading and then the list of stories.